prisma/seed.ts

```diff
@@ -300,6 +300,20 @@ seed('Default projects', async () => {
                     },
                 });
             }
+
+            const [goal] = allGoals;
+
+            // eslint-disable-next-line no-await-in-loop
+            await prisma.goalHistory.create({
+                data: {
+                    goalId: goal.id,
+                    subject: 'state',
+                    action: 'change',
+                    previousValue: 'InProgress',
```

Comment:
We need to write state.id here. If we want to view some meta about states later.